haystack/utils/hf.py

```diff
@@ -359,13 +359,15 @@ def __init__(
             tokenizer: Union[PreTrainedTokenizer, PreTrainedTokenizerFast],
             stream_handler: Callable[[StreamingChunk], None],
             stop_words: Optional[List[str]] = None,
+            component_info: ComponentInfo = ComponentInfo(),
```

Comment:
Also let's make this optional here. Once we make `component_info` optional in StreamingCallback let's make it optional in this init method.